tests/components/samsungtv/test_config_flow.py

```diff
@@ -926,12 +926,17 @@ async def test_autodetect_legacy(hass: HomeAssistant, remote: Mock):
 
 async def test_autodetect_none(hass: HomeAssistant):
     """Test for send key with autodetection of protocol."""
+    mock_remotews = Mock()
+    mock_remotews.__enter__ = Mock(return_value=mock_remotews)
+    mock_remotews.__exit__ = Mock()
+    mock_remotews.open = Mock(side_effect=OSError("Boom"))
+
```

Comment:
Here I couldn't simply patch `homeassistant.components.samsungtv.bridge.SamsungTVWS.open` because the tests are checking the constructor arguments:
https://github.com/home-assistant/core/blob/dbc445c2fa1feb2abaa7bd40c13196665ce468eb/tests/components/samsungtv/test_config_flow.py#L946-L949